src/components/AdvanceAnalyticsV2/AnalyticsV2Page.jsx

```diff
@@ -15,15 +15,22 @@ import Leaderboard from './tabs/Leaderboard';
 import Skills from './tabs/Skills';
 import { useEnterpriseAnalyticsAggregatesData } from './data/hooks';
 import { GRANULARITY, CALCULATION } from './data/constants';
+import { DEFAULT_GROUP } from './constants';
 import { formatTimestamp } from '../../utils';
+import { useAllFlexEnterpriseGroups } from '../learner-credit-management/data';
 
 const AnalyticsV2Page = ({ enterpriseId }) => {
   const [activeTab, setActiveTab] = useState('enrollments');
   const [granularity, setGranularity] = useState(GRANULARITY.WEEKLY);
   const [calculation, setCalculation] = useState('Total');
+  const [groupUUID, setGroupUUID] = useState('');
```

Comment:
Initially we will fetch data regardless of groups, correct?

Reply:
Yes, that is correct. empty value "" means no group selected.